tsconfig.json

```diff
@@ -12,5 +12,6 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "outDir": "dist"
-  }
+  },
+  "include": ["src/**/*"]
```

Comment:
Adding `scripts/postinstall.js` actually meant that `tsc` would pick it up, so I thought I'd limit it to `src` only.